lib/Db/Page.php

```diff
@@ -32,15 +34,21 @@ class Page extends Entity implements JsonSerializable {
 	protected ?string $lastUserId = null;
 	protected ?string $emoji = null;
 	protected ?string $subpageOrder = null;
+	protected ?bool $fullWidth = null;
 	protected ?int $trashTimestamp = null;
 
+	public function __construct() {
+		$this->addType('fullWidth', 'bool');
```

Comment:
We don't use `addType` in the constructors so far and I don't think it's necessary with typing in the class attributes. But not exactly sure about this. What was the reasoning behind adding it for you?

Reply:
It not works without this change :) Postgres returns some error, like it can't set 1 to field which expects bool

Reply:
That should be fine and the way go https://docs.nextcloud.com/server/latest/developer_manual/basics/storage/database.html#entities to ensure that database column types are properly mapped

Those types will be used then in the mapper https://github.com/nextcloud/server/blob/2fb0ca9cf7929b3527f2068e19a690c371191b72/lib/public/AppFramework/Db/QBMapper.php#L213-L236

These days it would actually be nicer to inherit the type from the type hints but that's another story for improvement in server.

Reply:
so, I guess we can keep it as is